### ISSUE TITLE
Image IDs of Test set images

### ISSUE THREAD

**Issue by YeahMao** [ibm-aur-nlp/PubLayNet#29]
Hi PubLayNet Team,

Thank you for sharing the dataset. I am wondering is there a Json file (or a list of image id) of the Test set provided?

**Comment by ajjimeno** [comment]
Hi YehMao,

  Currently, there is no public ground truth for the test set. The ground truth is being reserved for the following ICDAR competition:

  https://icdar2021.org/competitions/competition-on-scientific-literature-parsing

**Comment by YeahMao** [comment]
> Hi YehMao,
> 
> Currently, there is no public ground truth for the test set. The ground truth is being reserved for the following ICDAR competition:
> 
> https://icdar2021.org/competitions/competition-on-scientific-literature-parsing

Hi ajjimeno,

Thank you for your reply. 

Yes, I participated in this competition, and I am trying to make a submission. I can't pass the phase if I don't have the correct image IDs. Therefore, I am wondering how can I get the corresponding id of an image. For train and val, I saw the IDs by checking the "train.json" and "val.json" files. For the test set, is there an alternative way to find the image IDs? 

Thanks.

**Comment by ajjimeno** [comment]
Hi YeahMao,

  Thank you for pointing it out. We are preparing the file linking the file
names and ids and will release it soon.

**Comment by ajjimeno** [comment]
I have made available the ids from the github of the challenge:
https://github.com/ibm-aur-nlp/PubLayNet/blob/master/ICDAR_SLR_competition/test_ids.json

  Best regards,
  Antonio

**Comment by YeahMao** [comment]
Hi Antonio,

Thank you for the update.

I have another question regarding the ICDAR competition. Is there a ground
truth for the mini validation set (20 examples)?

I noticed that there is a "anntations.png" file provided which is converted
from the "samples.json" file. On the competition page, there is a host
submission that matches this Json file. Since the overall score of it is
not one out of one, I reckon it is just a baseline? Or should we regard the
annotations.png as a ground truth? Thank you in advance.

Best regards,
Tony

**Comment by ajjimeno** [comment]
Hi Tony,

  The annotations for those files should be available from the
development/validation set. Please let me know if you have any
issue finding them.

  Best regards,
  Antonio
